terraform/modules/localImage/main.tf

```diff
@@ -0,0 +1,12 @@
+resource "null_resource" "build_local_image" {
```

Comment:
There is an option in the docker provider to build the image. You should use that instead.

Reply:
I tried this option but it fails because it does not support having the `Dockerfile` located in a different folder than the context